js/apps/admin-ui/package.json

```diff
@@ -81,7 +81,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-dropzone": "^14.2.3",
-    "react-error-boundary": "^3.1.4",
+    "react-error-boundary": "^4.0.11",
```

Comment:
Now that this dependency is up-gradable, it can also be removed from the denylist in our [Dependabot configuration](https://github.com/keycloak/keycloak/blob/8b65e6727b9705d18c0e343f7409d516044929e0/.github/dependabot.yml#L37-L39), could you update that as well?